docs/blogs/kubecon_cfp.md

```diff
@@ -0,0 +1,45 @@
+# ElasticDL: Kubernetes-native Deep Learning System with Elastic Scheduling
+
+## Description
+
+In these years, Kubernetes becomes the fundamental infrastructure of machine
+learning. Kubeflow project provides some operators to run distributed machine
+learning jobs on Kubernetes. It uses gang scheduling policy. For example, a job
```

Comment:
In these solutions, Kubernetes plays the role to restart preempted pods.  However, such retrials often fail due to the same reason that caused the perception -- the lack of resources.  Couldn't maintain the constant number of worker pods, jobs fail.  Maintaining the constant number, it becomes gang scheduling.  Either case leads to low utilization of clusters.

Reply:
Good suggestion, updated according to the comments.